File: app/src/main/java/com/asmat/rolando/popularmovies/utilities/ViewUtils.java
package com.asmat.rolando.popularmovies.utilities;

import android.content.Context;
import android.content.res.Configuration;

/**
 * Created by rolandoasmat on 5/29/17.
 */

public class ViewUtils {

    public static int calculateNumberOfColumns(Context context) {
        int orientation = context.getResources().getConfiguration().orientation;
        if(orientation == Configuration.ORIENTATION_LANDSCAPE) {
            return 3;
        } else {
            return 2;
        }

    }
}

File: app/src/main/java/com/asmat/rolando/popularmovies/utilities/ViewUtils.kt
package com.asmat.rolando.popularmovies.utilities

import android.content.Context
import android.content.res.Configuration

object ViewUtils {

    /**
     * Determine the number of columns to use in the Movies Grid
     * depending on orientation.
     */
    fun calculateNumberOfColumns(context: Context): Int {
        val orientation = context.resources.configuration.orientation
        return if (orientation == Configuration.ORIENTATION_LANDSCAPE) {
            3
        } else {
            2
        }

    }
}
